### PR TITLE
feat(base): creates project base and server

### PR DESCRIPTION
**Pull request by wcthompson**
What: Initialized the node package for my simple api training!

Why: Step one of my 'build-your-own-api' training.

Details:
- Initialized the package and created the base structure with Lob's yeoman generator
- Installed Hapi and it's dependancies, wrote a basic server and config objects 
- Shrinkwrapped dependancies

### CODE REVIEW

File: lib/server.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+const Config = require('../config');
+
+const server = new Hapi.Server({
+  connections: {
+    router: { stripTrailingSlash: true }
+  }
+});
+
+server.connection({ port: Config.PORT });
```
Comment (elnaz) [inline lib/server.js]: add a new line after this

File: package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "sfmovies",
+  "version": "0.0.0",
+  "description": "Will's API for movies filmed in San Fransisco",
```
Comment (elnaz) [inline package.json]: typo

**Comment by elnaz** [comment]
LGTM!

File: lib/index.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+const Util = require('util');
+
+const Config = require('../config');
+const Movies = require('./server');
+
+Movies.start(() => {
+  Util.log(`Server start on port: ${Config.PORT}`);
+});
+
+
```
Comment (charlesbjohnson) [inline lib/index.js]: nit: remove the excess newlines

File: package.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  "version": "0.0.0",
+  "description": "Will's API for movies filmed in San Francisco",
+  "main": "lib/index.js",
+  "scripts": {
+    "enforce": "istanbul check-coverage --statement 100 --branch 100 --function 100 --lines 100",
+    "lint": "eslint .",
+    "release:patch": "changelog -p && git add CHANGELOG.md && git commit -m 'updated CHANGELOG.md' && npm version patch && git push origin && git push origin --tags",
+    "release:minor": "changelog -m && git add CHANGELOG.md && git commit -m 'updated CHANGELOG.md' && npm version minor && git push origin && git push origin --tags",
+    "release:major": "changelog -M && git add CHANGELOG.md && git commit -m 'updated CHANGELOG.md' && npm version major && git push origin && git push origin --tags",
+    "start" : "nodemon --ignore test/",
```
Comment (charlesbjohnson) [inline package.json]: nit: extra space

**Comment by charlesbjohnson** [comment]
LGTM

**Pull request merged**